build_scripts/build_runner.py

```diff
@@ -179,9 +179,28 @@ def run(self):
 
                 self.log.debug(completed_process.stdout)
             except subprocess.CalledProcessError as process_error:
-                self.log.error(process_error.stdout)
+                if self.stage == Stage.BUILD:
+                    self._parse_logs(process_error.stdout)
+                else:
+                    self.log.error(process_error.stdout)
                 raise
 
+    def _parse_logs(self, stdout):
+        self.log.info(stdout)
+        output = [""]
```

Comment:
Please add a message like:
The errors below were found in the output. See full log for details.